README.md

```diff
@@ -58,6 +58,34 @@ https://github.com/open-source-parsers/jsoncpp/wiki/Amalgamated-(Possibly-outdat
 ### The Meson Build System
 If you are using the [Meson Build System](http://mesonbuild.com), then you can get a wrap file by downloading it from [Meson WrapDB](https://wrapdb.mesonbuild.com/jsoncpp), or simply use `meson wrap install jsoncpp`.
 
+### The Bazel build system
```

Comment:
I'd like to keep `README.md` short and move as much as possible to our wiki? The reason is simple: To avoid doc-only changes in the source-code repo.

Reply:
Removed